app/src/js/components/SortableTable/SortableTable.js

```diff
@@ -10,9 +10,11 @@ import PropTypes from 'prop-types';
 import classNames from 'classnames';
 import omit from 'lodash/omit';
 import { useTable, useResizeColumns, useFlexLayout, useSortBy, useRowSelect, usePagination } from 'react-table';
+import { dispatch } from 'd3';
```

Comment:
We don't want to be importing dispatch from d3 because it's not the right method.

I think what might be easiest here is doing

`const dispatch = useDispatch()` in the component and importing useDispatch from `react-redux`